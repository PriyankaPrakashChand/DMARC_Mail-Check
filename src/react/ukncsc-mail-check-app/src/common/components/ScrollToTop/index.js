import { withRouter } from 'react-router-dom';
import ScrollToTop from './ScrollToTop';

export default withRouter(ScrollToTop);
