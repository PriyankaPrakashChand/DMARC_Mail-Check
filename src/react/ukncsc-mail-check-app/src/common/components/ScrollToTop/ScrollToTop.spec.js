import React from 'react';
import { render } from 'react-testing-library';
import ScrollToTop from './ScrollToTop';

describe('ScrollToTop', () => {
  let container;

  beforeEach(() => {
    ({ container } = render(
      <ScrollToTop>
        <p>some content</p>
      </ScrollToTop>
    ));
  });

  test('it should match the snapshot', () =>
    expect(container).toMatchSnapshot());
});
