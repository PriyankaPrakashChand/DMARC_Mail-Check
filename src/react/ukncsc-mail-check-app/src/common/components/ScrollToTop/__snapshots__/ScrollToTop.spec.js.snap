// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ScrollToTop it should match the snapshot 1`] = `
<div>
  <p>
    some content
  </p>
</div>
`;
